README.md

```diff
@@ -39,18 +40,34 @@ spark_ai.activate()  # active partial functions for Spark DataFrame
 
 You can also pass other LLMs to construct the SparkAI instance. For example, by following [this guide](https://python.langchain.com/docs/integrations/chat/azure_chat_openai):
 ```python
-from langchain.chat_models import AzureChatOpenAI
+from langchain.chat_models import ChatLiteLLM
 from pyspark_ai import SparkAI
 
-llm = AzureChatOpenAI(
-    deployment_name=...,
-    model_name=...
-)
+# using ChatLiteLLM with your selected models
+llm = ChatLiteLLM(model_name="gpt-4")
```

Comment:
nit: how about we only keep one `llm =` and comment out the others for clarity and executability?